### PR TITLE
feat: input & images

### PR DESCRIPTION
**Pull request by divyakoli1401**

### CODE REVIEW

File: input.css [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  }
+
+  /* radio button style */
+  
+  .radiobtn-input {
+    height: 25px;
+    width: 25px;
+    background-color: #eee;
+    cursor: pointer;
+  }
```
Comment (BelideNikhil) [inline input.css]: Combine them if you are having same properties on both!

Rest all is good!

File: input.css [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    max-width: 300px;
+    height: auto;
+  }
+
+  .round-image{
+    width: 100%;
+    max-width: 300px;
+    height: auto;
+    border-radius: 50%;
+  }
```
Comment (BelideNikhil) [inline input.css]: same here

**Review by BelideNikhil — APPROVED** [review]

File: css/buttons.css [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+/* float action icon */
+
+.btn-float-icon-container{
+    position: relative;
+}
+
+.btn-float-icon{
+ position: absolute;
+ z-index: 999;
+ bottom: 20px;
```
Comment (zabihshaik) [inline css/buttons.css]: Better to use rems instead of px

File: css/colors.css [hunk 4]
```diff
@@ -0,0 +1,23 @@
+:root{
+    --primary-color:rgb(196,85,168);
+    --secondary-color:rgb(222,205,245);
+    --complement-color:rgb(69,207,233);
+    --green-color: rgb(124,252,0);
+    --red-color:rgb(255, 0, 0);
+    --yellow-color:#ffcc00;
```
Comment (zabihshaik) [inline css/colors.css]: Better to be consistent in using either HEX values or rgb values to colors.

**Review by zabihshaik — COMMENTED** [review]
Everything looks Good. 
Just a couple of suggestions. Please check the comments.

File: input.css (hunk 1 above)
Comment (divyakoli1401) [reply input.css]: thank you! made the changes in code

File: css/colors.css (hunk 4 above)
Comment (divyakoli1401) [reply css/colors.css]: thank you for pointing it out changed it to rgb value.

File: css/buttons.css (hunk 3 above)
Comment (divyakoli1401) [reply css/buttons.css]: changed the unit to rem thank you

**Pull request merged**